common/modules/token_send/src/token_send.rs

```diff
@@ -23,6 +23,8 @@ pub trait TokenSendModule {
             function = ManagedBuffer::new();
         }
 
+        self.send().direct_with_gas_limit();
```

Comment:
why is this needed ?

Reply:
It isn't, I started updating this module, and eventually gave up and didn't use it in any of the contracts anymore